requirements/base.in

```diff
@@ -0,0 +1,8 @@
+# Core requirements for using this package
+-c constraints.txt
+
+edx-opaque-keys
+PyContracts
+sphinx
```

Comment:
I think that in this file we only should have the dependencies that were in setup.py before this change, namely:
PyContracts, edx-opaque-keys and xblock, the other two seems to me that should be in a doc.in file or something like that.

Reply:
done.